### PR TITLE
🐛 A form cannot be created without content #1527

### PR DESCRIPTION
**Pull request by ithiame**

### CODE REVIEW

File: src/lib/components/CmsDesign.svelte [hunk 1]
```diff
@@ -93,7 +93,7 @@
 				subject: contactForm.subject.replace(/{{([^}]+)}}/g, (match, p1) => {
 					return lowerVars[p1.toLowerCase()] || match;
 				}),
-				content: contactForm.content.replace(/{{([^}]+)}}/g, (match, p1) => {
+				content: contactForm.content?.replace(/{{([^}]+)}}/g, (match, p1) => {
```
Comment (coyotte508) [inline src/lib/components/CmsDesign.svelte:96]: revert

**Review by coyotte508 — APPROVED** [review]

**Comment by Tirodem** [comment]
Nah.
image
BTW, this 422 error introduction since few weeks bothered me more than reasonnable treshold and each fix seems to be complicated.
It works on edit to remove content but on creation it doesn't work after deploy.
Please take it again and don't submit it without test video + same on deploy & test on dev-bootik2.

File: src/routes/(app)/admin[[hash=admin_hash]]/form/new/+page.server.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const actions: Actions = {
 				title: z.string().trim().min(1).max(MAX_NAME_LIMIT),
 				target: z.string().trim().min(1).max(100),
 				subject: z.string().trim().min(1).max(100),
-				content: z.string().trim().min(1).max(MAX_CONTENT_LIMIT),
+				content: z.string().trim().min(1).max(MAX_CONTENT_LIMIT).default(''),
```
Comment (coyotte508) [inline src/routes/(app)/admin[[hash=admin_hash]]/form/new/+page.server.ts:30]:
```suggestion
				content: z.string().trim().max(MAX_CONTENT_LIMIT),
```

This is probably the fix

**Comment by ithiame** [comment]
https://github.com/user-attachments/assets/6e8b6325-80e7-4ab8-a1ab-b258e962684d

**Review by coyotte508 — APPROVED** [review]

**Pull request merged**